### ISSUE TITLE
using keyboard navigation you can still select a date even if is disabled

### ISSUE THREAD

**Issue by pictoru** [eternicode/bootstrap-datepicker#1365]
I've set the datepicker with daysOfWeekDisabled to disable Sundays and Saturdays and based on some other rules, using beforeShowDay I've disabled some other days too. Everything works fine using the mouse, instead, using keyboard I can select a disabled date. 

I think, if a date is set as disabled, the user should not be able to move the focus on it.

**Comment by pictoru** [comment]
I've found a quick fix, I'm sure it is not the right one but probably could help for some cases. 
In my project, I've changed the way I disable specific dates from using beforeShowDay to datesDisabled option (I don't remember if this option was available when I started the project).

in the keydown method on enter case I've added 
    focusDate = this.focusDate || this.dates.get(-1) || this.viewDate;

    var cls = this.getClassNames(focusDate);
    if ($.inArray("disabled", cls) > -1) {
        e.preventDefault();
        e.stopPropagation();
        break;
    }

... immediately after enter (13) case statement 

This fix will still let focusing the date, which can be useful for user, but will not select a disabled date on enter.
